### DISCUSSION TITLE
为什么从2.13更新到3.0-2 服务里的选项直接没了

### DISCUSSION THREAD

**Discussion by Rui-xing** [UnblockNeteaseMusic/luci-app-unblockneteasemusic#189] [Q&A]
![Z2T{)GDQV4GP%@ T}~269SE](https://user-images.githubusercontent.com/84909508/186718595-78254cfa-702b-4aab-a5d9-e96d18342d52.png)

**Comment by 1715173329** [comment]
您的 luci 不支持 3.x 版本。

**Reply by Rui-xing** [reply]
那怎么办，要升级什么呀

**Reply by Rui-xing** [reply]
我更新成OpenWrt R22.8.2 / LuCI Master (git-22.236.14286-082687d)这个版本了还是不行

**Reply by 1715173329** [reply]
基于 lua 的 2.x 版本仍然接受维护与更新，您不必追求所谓“最新”版本。
